proposals/lip-0048.md

```diff
@@ -31,36 +31,41 @@ Each chain can configure the token used to pay fees. On the Lisk mainchain, the
-| **General Constants** | | | |
-| `MODULE_NAME_FEE` | string | "fee" | Module name of the Fee module. |
-| `EVENT_NAME_GENERATOR_FEE_PROCESSED` | string | "generatorFeeProcessed" | Event name of the GeneratorFeeProcessed event. |
-| `EVENT_NAME_RELAYER_FEE_PROCESSED` | string | "relayerFeeProcessed" | Event name of the RelayerFeeProcessed event. |
-| `EVENT_NAME_INSUFFICIENT_FEE` | string | "insufficientFee" | Event name of the InsufficientFee event. |
-| `LENGTH_ADDRESS` | uint32 | 20 | The length of an address in bytes. |
-| `LENGTH_TOKEN_ID` | uint32 | 8 | The length of a token ID in bytes. |
-| **Configurable Constants** | | **Mainchain Value** | |
-| `MIN_FEE_PER_BYTE` | uint64 | 1000 | Minimum amount of fee per byte required for transaction validity. |
-| `TOKEN_ID_FEE` | bytes | `OWN_CHAIN_ID[0:1] + '00000000000000'` | Token ID of the token used to pay the transaction fees. |
+| Name                                 | Type   | Value                                  | Description                                                       |
+| ------------------------------------ | ------ | -------------------------------------- | ----------------------------------------------------------------- |
+| **General Constants**                |        |                                        |                                                                   |
+| `MODULE_NAME_FEE`                    | string | "fee"                                  | Module name of the Fee module.                                    |
+| `EVENT_NAME_GENERATOR_FEE_PROCESSED` | string | "generatorFeeProcessed"                | Event name of the GeneratorFeeProcessed event.                    |
+| `EVENT_NAME_RELAYER_FEE_PROCESSED`   | string | "relayerFeeProcessed"                  | Event name of the RelayerFeeProcessed event.                      |
+| `EVENT_NAME_INSUFFICIENT_FEE`        | string | "insufficientFee"                      | Event name of the InsufficientFee event.                          |
+| `LENGTH_ADDRESS`                     | uint32 | 20                                     | The length of an address in bytes.                                |
+| `LENGTH_TOKEN_ID`                    | uint32 | 8                                      | The length of a token ID in bytes.                                |
+| **Configurable Constants**           |        | **Mainchain Value**                    |                                                                   |
+| `MIN_FEE_PER_BYTE`                   | uint64 | 1000                                   | Minimum amount of fee per byte required for transaction validity. |
+| `TOKEN_ID_FEE`                       | bytes  | `OWN_CHAIN_ID[0:1] + '00000000000000'` | Token ID of the token used to pay the transaction fees.           |
+| `ADDRESS_FEE_POOL`                   | bytes  | undefined                              | Address of the fee pool.                                          |
```

Comment:
It would be more consistent to set the value to `None` instead of undefined.

Reply:
Right!